content/desktop/release-notes.md

```diff
@@ -24,11 +24,60 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.22.0
+
+{{< release-date date="2023-08-03" >}}
+
+{{< desktop-install all=true version="4.22.0" build_path="/117440/" >}}
+
+### Upgrades
+- [Compose v2.21.0](https://github.com/docker/compose/releases/tag/v2.21.0)
+- [Docker Engine v24.0.6](https://docs.docker.com/engine/release-notes/24.0/#2406)
+- [Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1). With Docker Scout, you can now:
+  - You can now manage temporary and cached files with `docker scout cache`.
+  - You can now manage environments with `docker scout environment`.
+  - You can now configure default organization with `docker scout config`.
+  - List packages of an image with their vulnerabilities with `docker scout cves --format only-packages`.
+  - You can enroll an organization with Docker scout with `docker scout enroll`.
+  - You can now stop, analyse and compare local file system with `docker scout cves --type fs`.
+- Wasm runtimes:
+  - wasmtime, wasmedge revision `d0a1a1cd`.
+  - slight and spin wasm `v0.9.0`.
+
+### New
+
+- Added support for new Wasm runtimes: wws and lunatic.
+- `docker init` now supports ASP.NET
+- Increase performance of exposed ports, e.g. with `docker run -p`.
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug where docker stats would hang when Docker Desktop was in Resource Saver mode.
+- Fixed a bug where disabling experimental features via the Docker Desktop Dashboard Settings would not fully disable the resource saver feature.
+- Fixed a bug where the **Containers list** action button where clipped.
+- containerd-integration: Fixed `failed to read config content` error when interacting with some images.
+- containerd-integration: Fixed building Dockerfiles with `FROM scratch` instruction when using the legacy classic builder (`DOCKER_BUILDKIT=0`).
+- containerd-integration: Fixed `mismatched image rootfs errors` when building images with legacy classic builder (`DOCKER_BUILDKIT=0`).
+- containerd integration: Fixed `ONBUILD` and `MAINTAINER` Dockerfile instruction
+- containerd integration: Fixed healthchecks.
+
+#### For Mac
+
+- All users on macOs 12.5 or greater will now have VirtioFS enabled by default. This can be reverted in the **General** tab.
```

Comment:
```suggestion
- All users on macOs 12.5 or greater now have VirtioFS turned on by default. This can be reverted in **Settings** in the **General** tab.
```